gctscript/modules/gct/errors.go

```diff
@@ -0,0 +1,15 @@
+package gct
+
+import (
+	"fmt"
+
+	objects "github.com/d5/tengo/v2"
+)
+
+// errorResponse is a helper function to apply error details to a return object
+// for better script side error handling
+func errorResponse(format string, a ...interface{}) (objects.Object, error) {
```

Comment:
You make a good point what I will do is check the data that goes into the function to actually contain something. Because if it is ever implemented and misconfigured then it should return an error as you suggest.